usaspending_api/download/tests/integration/test_download_status.py

```diff
@@ -13,6 +13,9 @@
 from usaspending_api.etl.award_helpers import update_awards
 from usaspending_api.download.filestreaming import csv_generation
 
+EXPECTED_COLUMNS_TRANSACTIONS = 275
```

Comment:
It would be better to tie the expected column counts to the ordered dictionaries in `usaspending_api/download/v2/download_column_historical_lookups.py`

Makes the tests more durable and accurate

Reply:
And entirely useless IMO. If you deleted a column you didn't intend to, the test would auto-correct and you wouldn't observe it. That would be a great example of a test (or at least part of a test; these do check other things) we would be better off deleting.

Reply:
> If you deleted a column you didn't intend to

I sure hope that the PR creator and the PR reviewer would catch that situation regardless of any tests.

The ordered dicts in the mapping file is what one expects to populate the headers for a complete download file, a test confirming that isn't useless.